src/index.html

```diff
@@ -4,10 +4,61 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+          rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="attr-req-value"
+    >
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/37e083f6-cd0c-4c05-9c5e-bcfd12667aca)
Please fix this in the whole file.